dart/dynamics/Joint.h

```diff
@@ -347,6 +354,16 @@ class Joint : public virtual common::Subject
   /// Get upper limit for position
   virtual double getPositionUpperLimit(size_t _index) const = 0;
 
+  /// Gets whether a generalized coordinate is cyclic. Returns true if and only
```

Comment:
This level of detail is not consistently covered in coding standards, but here are some anecdotal guidelines:

https://www.drupal.org/coding-standards/docs
> For most functions (see exceptions below), summary lines must start with a third person singular present tense verb, and they must explain what the function does. Example: "Calculates the maximum weight for a list."

http://babbage.cs.qc.edu/courses/cs101/Coding_Guidelines.html
> The first thing inside a javadoc block has to be a sentence (capitalize the first word and end with a period) that tells what the class, interface, or method does. The style for this sentence is to start with a present-tense verb with the name of the class, interface, or method as the implicit subject. For example, instead of saying "Interface Abc defines the methods that must ...", say "Defines the methods that must ...". There can be more than one sentence in the description, but early versions of javadoc limited the description to just one, and it's better to make your description brief rather than wordy.

http://docs.oracle.com/javase/7/docs/api/
Convention in Javadoc is similarly to have the `s`: